Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -333,15 +542,17 @@ private void ExtractFields(ObservableList<ExternalItemFieldBase> fields, string
             }).Result));
         }
 
-        private static ObservableList<ExternalItemFieldBase> AddFieldsValues(ListResult<FieldMetadata> entityFields, string entityType, Dictionary<string, List<string>> listnodes, Dictionary<string, List<string>> phases)
+        private ObservableList<ExternalItemFieldBase> AddFieldsValues(ListResult<FieldMetadata> entityFields, string entityType, Dictionary<string, List<string>> listnodes, Dictionary<string, List<string>> phases)
         {
             ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
+            List<Release> releases = GetReleases();
```

Comment:
Codacy found an issue: [Rename 'releases' which hides the field with the same name.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6361231)